tests/basic/sharedv4_svc_test.go

```diff
@@ -400,7 +400,20 @@ var _ = Describe("{Sharedv4SvcFunctional}", func() {
 
 							Step(fmt.Sprintf("get counters from node %v before %s", attachedNodeBefore.Name, failoverLog),
 								func() {
-									countersBefore = getAppCounters(apiVol, attachedNodeBefore, counterCollectionInterval)
+									activeKeys := 0
+									Eventually(func() bool {
+										countersBefore = getAppCounters(apiVol, attachedNodeBefore, counterCollectionInterval)
+										activeKeys = 0
+										for key := range countersBefore {
+											if countersBefore[key].active {
+												activeKeys++
+											}
+										}
+										return (activeKeys == numPods)
+									}, 3*time.Minute, 10*time.Second).Should(BeTrue(),
+										"number of active keys did not match %v/%vfor volume %v (%v) for app %v",
```

Comment:
need space before "for volume %v"

There is no need to log the values of activeKeys and numPods since gomego will print those in the failure message But we should probably log the entire countersBefore map to help debug the failure.

Reply:
updated.